v2.0/select-clause.md

```diff
@@ -0,0 +1,451 @@
+---
+title: Simple SELECT Clause
+summary: The Simple SELECT clause loads or computes data from various sources.
+toc: false
+---
```

Comment:
Since we're renaming this file, we need to add a client-side redirect to the front-matter:

```
redirect-from: select.html
```